### PR TITLE
Add background-color, box-shadow and padding to .ClusterStatus

### PR DESCRIPTION
**Pull request by chenhunghan**
Improve the style of the modal pop-up when cluster is disconnected (or having troubles connecting).

Before
Screenshot 2021-08-10 at 09 22 47

After
Screenshot 2021-08-10 at 09 21 08

Signed-off-by: Hung-Han (Henry) Chen <chenhungh@gmail.com>

### CODE REVIEW

File: src/renderer/components/cluster-manager/cluster-status.scss [hunk 1]
```diff
@@ -27,6 +27,9 @@
   margin: auto;
   text-align: center;
   z-index: 1;
+  background: var(--contentColor);
+  box-shadow: 0 0 16px var(--boxShadow);
```
Comment (chenhunghan) [inline src/renderer/components/cluster-manager/cluster-status.scss]: Not sure shadow has proper values, please advise @aleksfront

**Comment by chenhunghan** [comment]
Not sure why CI `Check License Header / css (pull_request)` failed.
I tried locally `addlicense -check -l mit -c "OpenLens Authors" src/**/*.?css` seems to pass.

**Comment by aleksfront** [comment]
* Made the `ClusterStatus` view full-screen so it doesn't "flow" over the dashboard.
* Moved styles to css module
* Fixed layout bugs

fullscreen error
visible scrollbar


@chenhunghan Please take a look ^

**Comment by chenhunghan** [comment]
@aleksfront this looks better, thanks.

**Comment by chenhunghan** [comment]
@jakolehm can you take a look at the styling changes of cluster status elements?

**Review by jakolehm — APPROVED** [review]
LGTM

**Pull request merged**